hubble/src/indexer/dummy/config.rs

```diff
@@ -0,0 +1,24 @@
+use color_eyre::eyre::Report;
+use sqlx::PgPool;
+use unionlabs::aptos::block_info::BlockHeight;
+
+use super::dummy::{DummyContext, DummyFetcherClient};
+use crate::indexer::{api::IndexerId, Indexer};
+
+#[derive(Clone, Debug, serde::Deserialize)]
+pub struct Config {
+    pub indexer_id: IndexerId,
+    pub start_height: BlockHeight,
+}
+
+impl Config {
+    pub async fn build(self, pg_pool: PgPool) -> Result<Indexer<DummyFetcherClient>, Report> {
```

Comment:
Does this need to be async?

Reply:
want to have it consistent with other indexers (where we do some async initialization). 

ultimately (once all indexers are migrated) I'l remove the async from all indexers.